pavanh007-initial-commit

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "Pavan H",
+    "type": "personal",
+    "displayStatsPublic": true,
```

Comment:
In addition to the above requested changes, please remove `type`, & `displayStatsPublic` keys from your `json` as we no longer need them. 

[Edit Your File Here](https://github.com/pavanh007/LinkFree/edit/pavanh007-patch-1/pavanh007-initial-commit)

```suggestion
```